mchf-eclipse/drivers/audio/freedv_mchf.c

```diff
@@ -199,9 +203,11 @@ void fdv_print_ber()
 }
 
 void fdv_clear_display()
-{
+{ char clr_string[freedv_rx_buffer_max];
+
+  snprintf(clr_string,freedv_rx_buffer_max,"                                                                           ");
```

Comment:
Maybe because some likes to count spaces, since it is a hard and difficult task?
Maybe because it did the job?
Maybe because you didn't sent a pull request with the change :-)